autofit/graphical/messages/abstract.py

```diff
@@ -160,7 +176,7 @@ def __str__(self) -> str:
             return f"{type(self).__name__}(\n{param_strs})"
         else:
             param_strs = ', '.join(
-                attr + '=' + 
+                attr + '=' +
```

Comment:
Prefer f strings

Reply:
Yeah f strings are nice, but in this case it would require adding an additional generator comprehension to get it to work, which I think is less nice and readable.